doc/dev/peering.rst

```diff
@@ -256,4 +256,5 @@ The high level process is for the current PG primary to:
 State Model
 -----------
 
-.. graphviz:: peering_graph.generated.dot
+.. image:: peering_graph.generated.svg
+   :target: peering_graph.generated.svg
```

Comment:
please remove this `:target:` line. because sphinx will copy the file to the output directory with the right path. and with this line, the link url will be `$baseurl/dev/peering/peering_graph.generated.svg`, which will lead us to a 404, see http://docs.ceph.com/ceph-prs/19346/dev/peering/peering_graph.generated.svg, but not http://docs.ceph.com/ceph-prs/19346/_images/peering_graph.generated.svg.